src/app/calendar/parcelCalendarFunctions.ts

```diff
@@ -1,30 +1,20 @@
 import { Schema } from "@/database_utils";
-import supabase from "@/supabaseClient";
 import { CalendarEvent } from "@/components/Calendar/Calendar";
 
 export type ParcelWithClientName = Schema["parcels"] & { clients: { full_name: string } | null };
 
+export type ParcelsWithExtraFields = ParcelWithClientName & {
+    collection_centres: { name: string } | null;
+};
+
```

Comment:
This is some comments I received from my branch, but seems like it is more readable to define a `type  CollectionCentres = {
    collection_centres: { name: string } | null;
};` and chuck it in the type now